framework/doc/content/source/actions/Action.md

```diff
@@ -25,3 +113,13 @@ overridden. Both
 the `ContactAction` in the contact module, and `PorousFlowActionBase` in the
 porous flow module provide examples of overriding this method. For the reasons
 behind why this must be done in the action system, please see [RelationshipManager.md#rm_action].
+
+## Troubleshooting Actions
+
+To see the list of actions as they execute, use `Debug/show_actions=true`:
+
+```
+[Debug]
+  show_actions = true
+[]
+```
```

Comment:
Maybe having a section of advanced features, such as

1. Actions' lifetime is the entire simulation. They get destructed when exiting the run.
2. A particular Action can be obtained through `ActionWarehouse` with functions like `getAction`, `getActions`, etc.
3. Action can be inserted by another Action (action of action) but the newly inserted action must registered tasks happening after the task where insertion happens.

Reply:
I don't know how to write such a section well without having the context of when those details become important. I guess at the very list I could add an "Advanced Notes" section at the end with those 3 bullet points. Then hopefully someone would write some context in there later.

Reply:
Yeah, that is fine. I did not think of how this should be structured very carefully. Just wanted to point out few things that are important with our experiences in Griffin.

Reply:
Ok, I wrote the section - please let me know if I've translated your 3rd bullet correctly.